src/go/rpk/pkg/tuners/redpanda_checkers_test.go

```diff
@@ -7,6 +7,8 @@
 // the Business Source License, use of this software will be governed
 // by the Apache License, Version 2.0
 
+// +build linux
```

Comment:
nice! Thanks for this. I think we should have a github action that enables testing then on the mac os builds.